docs/platform/howto/technical-emails.rst

```diff
@@ -14,6 +14,9 @@ Set up email notifications
 
 #. Click **Save changes**. 
 
+.. note::
```

Comment:
I think this would be better as part of the introduction (the first paragraph) and slightly reworded as: "If you don't add a technical email for a project, email notifications will still be sent to users with admin and operator roles."

Reply:
Thanks for the review and suggestion. I have made the changes to the new commit.